tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
This is an issue that most the macros have. I couldn't decide what the default value is if all the params are undefined, so most return undefined. I'm seeing that lodash returns empty arrays in most cases, so maybe all the macros need this change.